scenarios/kubernetes_bazel.py

```diff
@@ -71,10 +71,15 @@ def main(args):
             res = 1
         else:
             try:
-                check(
-                    'bazel', 'run', '//:push-build', '--',
-                    '%s/%s' % (args.gcs, version)
-                    )
+                gcs_location = '%s/%s' % (args.gcs, version)
+                check('bazel', 'run', '//:push-build', '--', gcs_location)
```

Comment:
This seems extremely specific to a particular use case, and I would rather leave this scenario generic and applicable to any bazel scenario (isn't that the intent?) 

Is there any way we could make the push-build rule do this for us?